code/ui/.storybook/preview.jsx

```diff
@@ -150,6 +151,37 @@ export const decorators = [
       }
     }
   },
+  /**
+   * This decorator shows the current state of the arg named in the
+   * parameters.withRawArg property, by updating the arg in the onChange function
+   * this also means that the arg will sync with the control panel
+   *
+   * If parameters.withRawArg is not set, this decorator will do nothing
```

Comment:
Note this decorator also provides an `onChange` arg that updates the real value of the nominated ("`withRawArg`") when the callback is called. So I wonder if the name `withRawArg` might not tell the whole story.

I can't think of a great name though 🤔

Reply:
Me neither, let's revisit if we come up with something better.